mitmproxy/addons/view.py

```diff
@@ -441,7 +441,11 @@ def resolve(self, spec: str) -> typing.Sequence[mitmproxy.flow.Flow]:
 
     @command.command("view.create")
     def create(self, method: str, url: str) -> None:
-        req = http.HTTPRequest.make(method.upper(), url)
+        try:
+            req = http.HTTPRequest.make(method.upper(), url)
+        except ValueError as e:
+            ctx.log.error(e)
```

Comment:
I think the right approach would be to raise a CommandError, as e.g. done [here](https://github.com/MatthewShao/mitmproxy/blob/633bc350d225eb02c7ab0aef6bae57c4def560e1/mitmproxy/addons/view.py#L439) (@cortesi, is that right?):
```python
raise exceptions.CommandError("Cannot create flow: %s" % e) from e
```